lib/kubernetes-deploy/kubernetes_resource/pod.rb

```diff
@@ -120,5 +140,69 @@ def display_logs
 
       @already_displayed = true
     end
+
+    class Container
+      STATUS_SCAFFOLD = {
```

Comment:
Sorta. It's used to provide a reliable structure so that I don't have to be nil checking everywhere.